src/components/Badge.astro

```diff
@@ -0,0 +1,18 @@
+---
+
+const { type = "default", text } = Astro.props;
+
+const coloursMap = {
+  experimental: 'bg-surface-accent-light text-content-accent border-outline-opaque',
+  GET: 'border-green-400 bg-green-100 text-content-primary',
+  POST: 'border-cyan-400 bg-cyan-100 text-content-primary',
+  PUT: 'border-orange-400 bg-orange-100 text-content-primary',
+  DELETE: 'border-red-400 bg-rose-100 text-content-primary',
+  default: 'text-content-primary border-outline-opaque bg-surface-tertiary'
+}
+
+---
+
+<span class=`${ coloursMap[type] } inline-flex text-xs px-2 py-0.5 rounded-full border`>
```

Comment:
@alexBinns are we missing a text style here? The design looks like it has `type-caption-2`